lib/services/role.go

```diff
@@ -86,7 +86,7 @@ func RoleNameForCertAuthority(name string) string {
 }
 
 // NewAdminRole is the default admin role for all local users if another role
-// is not explicitly assigned (Enterprise only).
+// is not explicitly assigned (this role applies to all users in OSS version).
```

Comment:
Do we happen to have an example of the OSS `admin` role, I think it would be handy to have the defaults so I can add them to the docs.

Reply:
I think the `admin` role looks something like this:

```yaml
kind: role
metadata:
  name: admin
spec:
  allow:
    kubernetes_groups: [ '{{internal.kubernetes_groups}}' ]
    logins: [ '{{internal.logins}}' ]
    node_labels:
      '*': '*'
    rules:
    - resources: [ role ]
      verbs: [ list, create, read, update, delete ]
    - resources: [ auth_connector ]
      verbs: [ list, create, read, update, delete ]
    - resources: [ session ]
      verbs: [ list, read ]
    - resources: [ trusted_cluster ]
      verbs: [ list, create, read, update, delete ]
    - resources: [ event ]
      verbs: [ list, read ]
  deny:
    logins: null
  options:
    cert_format: standard
    forward_agent: true
    max_session_ttl: 30h0m0s
    port_forwarding: true
```